CorsixTH/Lua/objects/surgeon_screen.lua

```diff
@@ -38,6 +39,71 @@ object.orientations = {
   },
 }
 
+-- Animation numbers below correspond to operating theatre uses only.
+-- See screen.lua for everything else.
+object.usage_animations = {
+  north = {
+    --[[ unused
+    begin_use = {
+      scrubs_on  = 2786,
+      scrubs_off = 2788
+    },
+    ]]--
+    in_use = {
+      Surgeon = {
+        scrubs_on_t1 =  2780,
+        scrubs_on_t2 =  2782,
+        scrubs_on_t3 =  2784,
+        scrubs_off_t1 = 2790,
+        scrubs_off_t2 = 2792,
+        scrubs_off_t3 = 2794,
+        scrubs_off_t4 = 2796,
+      },
+      ["Standard Male Patient"] = {
+        gown_on  = 4760,
+        gown_off = 4768,
+      },
+      ["Standard Female Patient"] = {
+        gown_on  = 4762,
+        gown_off = 4770,
+      },
+    },
+  },
+}
+
+-- Set markers for all animations involved.
+-- TODO better define these
+local animation_numbers = {
+  2780, -- SURGEON
+  2782, -- SURGEON
+  2784, -- SURGEON
+  --2786,  unused SURGEON?
+  --2788,  unused SURGEON?
+  2790, -- SURGEON
+  2792, -- SURGEON
+  2794, -- SURGEON
+  2796, -- SURGEON
+  4760, -- SURGEON PAT
+  4762, -- SURGEON PAT
+  4768, -- SURGEON PAT
+  4770, -- SURGEON PAT
+}
+
+local anim_mgr = TheApp.animation_manager
+local kf1, kf2, kf3 = {-1.2, -1.2}, {-0.9, -0.8}, {-0.7, -0.6}
```

Comment:
NB `{-1.2, -1.2}` is slightly off but haven't managed to improve on it.

Reply:
`0.1` is about 3 pixels vertically and 6 pixels horizontally. If you append another digit or switch to pixels positions, you can point at the intermediate pixels as well.